### PR TITLE
added new member "saumya

### PR DESCRIPTION
**Pull request by prachinandi**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by prachinandi** [comment]
added new member "Saumya"

File: public/team.html [hunk 1]
```diff
@@ -612,6 +612,23 @@ <h3>Mohit Madhav Rao</h3>
                         </div>
                     </div>
                
+                    <div class="col-md-6 col-lg-3 mb-4">
+                        <div class="team-member">
+                            <figure>
+                                <ul class="social">
+                                    <li><a href="#"><span class="icon-github"></span></a></li>
```
Comment (DesignrKnight) [inline public/team.html:619]: Is this line needed?

Comment (prachinandi) [reply public/team.html:619]: she is not having github account, incase she makes one will add in future so didn't delete the line😁

Comment (DesignrKnight) [reply public/team.html:619]: But won't that show a blank icon presently?

**Review by DesignrKnight — APPROVED** [review]

**Pull request merged**